server/protocols/jmap/pom.xml

```diff
@@ -80,6 +80,10 @@
             <groupId>${project.groupId}</groupId>
             <artifactId>apache-mime4j-dom</artifactId>
         </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>james-mdn</artifactId>
```

Comment:
mdn is a bad project name : when looking to the web for mdn, you find everything but the right RFC.
Please use either the complete name "Message Disposition Notification" or the functional description "Message Receipts"

Reply:
Sorry but I'm :-1: ...

"Receipts" means nothing to me. This term doesn't even appear in the codebase. And "receipt" have only 3 match on the RFC page. Seems like a usage of MDNs but not a good qualification of it.

Honestly, **james-message-disposition-notification** seems ridiculous to me... It's as if we had **james-sender-policy-framework**, **james-server-json-metadata-application-protocol**, **james-server-simple-messaging-transfer-protocol**, etc... It don't seems to be a project convention to me...

If I search **RFC MDN** my first page is https://tools.ietf.org/html/rfc3798 confirming that **mdn** is a better fit.

Furthermore, this naming had already been validated in a previous review...

Also with such a change, we might brake external potential usage...

Reply:
> Furthermore, this naming had already been validated in a previous review...

Not a valid argument, it's very usual to miss things during review

> If I search RFC MDN my first page is https://tools.ietf.org/html/rfc3798 confirming that mdn is a better fit.

Ok but nobody prefixes searchs by RFC without any advice to do so

> Honestly, james-message-disposition-notification seems ridiculous to me... It's as if we had james-sender-policy-framework, james-server-json-metadata-application-protocol, james-server-simple-messaging-transfer-protocol, etc... It don't seems to be a project convention to me...

Your examples are interesting : the first search results for SPF, JSON and SMTP gives me a link to sensible information. MDN doesn't. If you try to search "mdn mail" because James is about mails, you don't find the right information neither. So your argument doesn't hold IMO.

Reply:
> Ok but nobody prefixes searchs by RFC without any advice to do so

Me I do... It refine the search quitte well ....

> Your examples are interesting : the first search results for SPF, JSON and SMTP gives me a link to sensible information. MDN doesn't. If you try to search "mdn mail" because James is about mails, you don't find the right information neither. So your argument doesn't hold IMO.

I don't see why we should do something different than in the general case here...

Maybe we need RFC number in the **pom** description, but I don't think a complicated and long artifact ID is the solution to a lack of knowledge.